.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
Can we add the whole `.idea` folder to the global .gitignore file?
Those files probably shouldn't be in the repo in the first place (remove them form PR)